typhos/tests/test_utils.py

```diff
@@ -276,5 +276,12 @@ def test_take_top_level_widget_screenshots(qtbot: pytestqt.qtbot.QtBot):
     widget = QWidget()
     qtbot.addWidget(widget)
     screenshots = list(utils.take_top_level_widget_screenshots(visible_only=False))
```

Comment:
A potentially dumb question.  Why is this generating any QMenus at all?  A naive (and honestly idealistic) reading of this test is that we're creating a single `QWidget`, then taking a screenshot of it.  Knowing that widgets aren't really getting cleaned up between tests, I can see why QMenus get involved, but it shouldn't generate any... right?

Reply:
I have repeated this ~dumb~ question to myself many times over the course of this PR.

I have no answers 😦 
That said, there's one thing I didn't try: what if we load up gammaray when this test fails and we poke around with it there? It'd be worthwhile I think